src/diffusers/pipelines/wuerstchen/pipeline_wuerstchen.py

```diff
@@ -210,7 +210,9 @@ def encode_prompt(
             # For classifier free guidance, we need to do two forward passes.
             # Here we concatenate the unconditional and text embeddings into a single batch
             # to avoid doing two forward passes
-            return text_encoder_hidden_states, uncond_text_encoder_hidden_states
+            text_encoder_hidden_states = torch.cat([text_encoder_hidden_states, uncond_text_encoder_hidden_states])
+
```

Comment:
We can't do this - see Sayak's comment above. Let's make sure we `return text_encoder_hidden_states, uncond_text_encoder_hidden_states` so that the API stays consistent

Reply:
sorry! ok then reverting!